docs/architecture.md

```diff
@@ -42,10 +42,29 @@ The frontend will host the internal admin interface. The current application is
 
 ## Additional packages
 
+### HTTP Signature Utils
+
+The `http-signature-utils` package includes helper functions for
+
+- loading Ed25519 keys from file or creating them
+- generating JWKs from Ed25519 keys
+- creating HTTP signature headers
+- validate and verify HTTP signature headers
+
+Additionally, the package includes an app that generates HTTP digests and signatures for Postman.
```

Comment:
Do we have a link to the Postman collection anyway?

Reply:
We should link it in multiple places. I'll link it here, too.